src/lib.rs

```diff
@@ -0,0 +1,34 @@
+pub mod raw_rtde {
```

Comment:
Stutter is unnecessary. The library is already `rtde`. There's no need for modules to be called `rtde_this`, `rtde_that`. Nobody wants to write `ur_rtde::raw_rtde::fake_raw_rtde`. That should simply be `ur_rtde::raw::fake`.